CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog], and this project adheres to [Semantic Versioning].
 
 Releases prior to 7.0 has been removed from this file to declutter search results; see the [archived copy](https://github.com/dipdup-io/dipdup/blob/8.0.0b5/CHANGELOG.md) for the full list.
+## [8.0.0] - 2024-09-28
```

Comment:
Please, follow the current document format (Keep a Changelog). Example for unreleased section: https://github.com/dipdup-io/dipdup/pull/1102/files

Reply:
done!